packages/gatsby-plugin-cms/package.json

```diff
@@ -11,8 +11,7 @@
   "browser": "src/index.ts",
   "sideEffects": false,
   "scripts": {
-    "build": "tsc",
-    "prepare": "cross-env NODE_ENV=production yarn run build",
+    "build": "cross-env NODE_ENV=production tsc",
```

Comment:
This change was previously made in another PR, right? Is there more changes in this PR that aren't related to the source collections feature?

Reply:
Yes, I was waiting for it to be merged so I can rebase :)